run_tests.py

```diff
@@ -27,6 +27,7 @@ def is_included(backend):
     c = is_included("c")
     wat = is_included("wat")
     run = is_included("run")
+    run_with_dbg = is_included("run_with_dbg")
```

Comment:
Let's add a test here that we are running on Linux. Otherwise let's not run this test, since it doesn't work yet on other platforms.